resource/subfunctions/started_task.md

```diff
@@ -0,0 +1,276 @@
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"TSTTSKEL",
+      "class":"STARTED",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "info":[
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RALTER  STARTED             (TSTTSKEL)  STDATA   (TRUSTED     (YES))",
+          "messages":[
+            "IRR52148I Warning: A value for USER should be specified in STDATA.",
+            "IRR52149I Warning: STARTED profiles should have (or match) names with two qualifiers.",
+            "ICH11009I RACLISTED PROFILES FOR STARTED WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED."
+          ]
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.extract_started_task()`
+
+```python
+  def extract_started_task( self, started_task_name: str ) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+Extract an existing **Started Task** profile in the **STARTED** class.
+
+#### 📥 Parameters
+* `started_task_name`<br>
+  The name of the **Started Task** profile being defined to the **STARTED** class.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Trait dictionary** of the values of the traits extracted from the **STDATA** segment of the Resource Profile or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.extract_started_task("TSTTSKEL")
+{'user': '', 'group': '', 'trusted': 'yes', 'privileged': None, 'trace': None}
+```
+
+###### Trait Dictionary as JSON
+```json
+{
+  "user": "",
+  "group": "",
+  "trusted": "yes",
+  "privileged": None,
+  "trace": None
+}
+```
+
+## `ResourceAdmin.delete_started_task()`
+
+```python
+  def delete_started_task( self, started_task_name: str ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def delete_started_task(self, started_task_name: str) -> Union[dict, bytes]:
```

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.